pkg/cobrautl/error.go

```diff
@@ -12,13 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package command
+package cobrautl
```

Comment:
Non blocking suggestion: Are any of these exit codes really specific to cobra? This is more like exit util helper package if I understand correctly?

Reply:
They are not indeed. But all our 'binaries' run on cobra as a app-framework. I didn't found it worth to split completely.